.github/workflows/static-code-analysis.yml

```diff
@@ -7,10 +7,6 @@ on:
     branches:
       - dev
 
-defaults:
```

Comment:
See my notes about the workflow here: https://github.com/Lombiq/Utility-Scripts/pull/28#discussion_r1052640912 However, do we actually need this in this repo? Since it's already under OSOCE, this analysis should run.

Reply:
Yes, because OSOCE only runs PS Core analysis, but this workflow runs both Windows PS and PS Core. This makes sure that Invoke-Analyzers (and the custom analyzer rules) comply with Windows PS, otherwise consumer repositories won't be able to run analysis on it.

Reply:
I see, so for this repo we actually have to run a superset of analysis happening in OSOCE otherwise.